app/TaskManager/Observers/TaskObserver.php

```diff
@@ -21,4 +23,20 @@ public function created(Task $task)
     {
         Notification::send($this->getRecipients($task->taskable(), $task->created_by), new TaskCreated($task, $task->creator));
     }
+
+    /**
+     * Handle the task status "updating" event.
+     *
+     * @param  \App\Task $task
+     * @return void
+     */
+    public function updating(Task $task)
+    {
+        $creator_id = $task->creator->id;
+        $updater_id = Auth::id();
+
```

Comment:
use camelCase for variable name

Reply:
@Hasnayeen I squashed this camelCase commit with the previous commit. Please check,